modules/recon/companies-multi/shodan_org.py

```diff
@@ -41,9 +41,7 @@ def module_run(self, companies):
                                 self.insert_hosts(host=hostname, ip_address=port['ip_str'])
                         except KeyError:
                             self.insert_ports(ip_address=ipaddr, port=port['port'], protocol=port['transport'])
-                        print(rec_count)
-                        print(total_results)
-                        print(page)
+
```

Comment:
I think we should bump the version on this because we are changing the output of the module. Folks might see the print statement, notice it is out of place, and file an issue since the change doesn't notify them of an upgrade.

Reply:
Ya those print statements were me debugging and should have never been in there. I updated the version to 1.1 should be good to go.